tests/fast-integration/test/fixtures/commerce-mock/index.js

```diff
@@ -58,9 +58,31 @@ const lastorderFunctionYaml = fs.readFileSync(
   }
 );
 
+const lastorderCentralApplicationGatewayFunctionYaml = fs.readFileSync(
+    path.join(__dirname, "..", "..", "..", "application-connectivity-2-test", "./lastorder-function.yaml"),
+    {
+      encoding: "utf8",
+    }
+);
+
+const lastorderCentralApplicationGatewayAndCompassFunctionYaml = fs.readFileSync(
+    path.join(__dirname, "..", "..", "..", "application-connectivity-2-compass-test", "./lastorder-function.yaml"),
+    {
+      encoding: "utf8",
+    }
+);
+
 const commerceObjs = k8s.loadAllYaml(commerceMockYaml);
 const applicationObjs = k8s.loadAllYaml(applicationMockYaml);
 const lastorderObjs = k8s.loadAllYaml(lastorderFunctionYaml);
+const lastorderCentralApplicationGatewayObjs = k8s.loadAllYaml(lastorderCentralApplicationGatewayFunctionYaml);
+
+function prepareLastorderCentralApplicationGatewayAndCompassObjs(appName) {
+  return k8s.loadAllYaml(
+      lastorderCentralApplicationGatewayAndCompassFunctionYaml
+          .toString()
+          .replace('%%APP_NAME%%', appName));
+}
```

Comment:
I think we could reuse the `.../fixture/commerce-mock/lastorder-function.yaml` and contents of `lastorderCentralApplicationGatewayObjs` which is loaded into obj anyway. 

Maybe we can reuse the line from `commerce-mock/lastorder-function.yaml` and simply override the `GATEWAY_URL` env or somehow construct the url to be placed in yaml during runtime?

Maybe additional function `patchGatewayURL()` or `patchLastorderFunctionGatewayURL()` that replaces, e.g., `%%GATEWAY_URL%%`, within the `commerce-mock` yaml, depending on the test scenario?

What would you say?

Reply:
Yup, I've done something like this. Will you take a look?